### PR TITLE
first stub

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Akka Platform Operator Charts
+
+This repository contains Helm charts for the [Akka Platform](https://developer.lightbend.com/docs/akka-platform-guide/).
```
Comment (patriknw) [inline README.md]: I think you can link to `[Akka Platform Operator](https://developer.lightbend.com/docs/akka-platform-guide/deployment/)`, where we will have an overview page.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Add this repository using the following commands:
+
+```
+helm repo add akka-operator-helm https://lightbend.github.io/akka-operator-helm/ 
+helm repo update
+```
+
+Install the operator (requires commercial license):
+```
+helm install akka-operator
```
Comment (patriknw) [inline README.md]: let's avoid too many instructions scattered at different places, link to the guide

**Comment by patriknw** [comment]
I misunderstood one thing when we talked about it. You moved the CRD here. That will be inconvenient during development of new features in the operator, since that often involves changes to the CRD. Different PRs to review and so on.

I thought there would be a publishing step that would copy the CRD from akka-platform-operator to here. Part of the Make (or something). To simplify that build we can assume that both repos are cloned with same parent directory, i.e. `../akka-platform-operator`

**Comment by patriknw** [comment]
and, the CRD url should be versioned v1, v2

File: README.md (hunk 1 above)
Comment (andreaTP) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (andreaTP) [reply README.md]: done

**Comment by andreaTP** [comment]
> I misunderstood one thing when we talked about it. You moved the CRD here. That will be inconvenient during development of new features in the operator, since that often involves changes to the CRD. Different PRs to review and so on.
> 
> I thought there would be a publishing step that would copy the CRD from akka-platform-operator to here. Part of the Make (or something). To simplify that build we can assume that both repos are cloned with same parent directory, i.e. `../akka-platform-operator`

Got it, let's keep here the `latest` version, I will automate the update from the private repo.

**Comment by andreaTP** [comment]
> and, the CRD url should be versioned v1, v2

done

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```
Comment (andreaTP) [inline LICENSE:1]: cc. @sebastian-alfers @JustinPihony 

added the License

File: README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Akka Platform Operator Charts
+
+This repository contains Helm charts for the [Akka Platform Operator](https://developer.lightbend.com/docs/akka-platform-guide/deployment/).
+
+## Maintenance
```
Comment (andreaTP) [inline README.md:5]: cc. @sebastian-alfers @JustinPihony 

Maintenance as Console chart (this repo have the same purpose):
https://github.com/lightbend/console-charts#maintenance

File: README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Enterprise Suite Team <es-all@lightbend.com>
+
+## License
+
+Copyright (C) 2018 Lightbend Inc. (https://www.lightbend.com).
```
Comment (patriknw) [inline README.md]: 2021

**Review by patriknw — APPROVED** [review]
LGTM

File: README.md (hunk 5 above)
Comment (andreaTP) [reply README.md]: right!

**Pull request merged**